README.md

```diff
@@ -0,0 +1,86 @@
+# auxtools
+
+auxtools is a Rust-based successor to the [Extools](https://github.com/MCHSL/extools) project.
+
+Currently, it implements some of the features such as the debug server, but is planned to supplant most functionality (maptick, etc.).
+
+## Features
+Debug Server - Working with SpaceManiac's [SpacemanDMM](https://github.com/SpaceManiac/SpacemanDMM), Auxtools interfaces with the debugger frontend to provide debugger information and management of breakpoints.
+
+*Dissassembly*  - When currently stopped at a breakpoint, enter the `#dis` command into the Debug Console in VSC to see the DM bytecode for the current proc. You can also provide it any arbitrary proc path like `#dis /mob/proc/Life`.
+
+## Dependencies
+
+*These instructions were taken directly from tgstation's [rust-g](https://github.com/tgstation/rust-g) documentation.*
```

Comment:
We have some instructions at https://github.com/willox/auxtools/wiki/Compiling if you want to compare them.